ddtrace/contrib/fastapi/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""
+The FastAPI__ integration is provided by a ASGI middleware.
+
+To configure tracing manually::
+
+    from ddtrace.contrib.asgi import TraceMiddleware
+
+    app = FastAPI()
+    app.add_middleware(TraceMiddleware)
```

Comment:
It could also show how to custom the name of the apm service
```python
app.add_middleware(TraceMiddleware, service="the_name_of_my_apm_service")
```